common/ux-alignment.cpp

```diff
@@ -46,10 +47,13 @@ bool is_gui_aligned(GLFWwindow *win)
         ImGui::PopStyleVar();
 
         ImGui::Render();
+        ImGui_ImplOpenGL3_RenderDrawData(ImGui::GetDrawData());
 
         glfwSwapBuffers(win);
 
-        ImGui_ImplGlfw_NewFrame(1.f);
+        //ImGui_ImplOpenGL3_NewFrame();
```

Comment:
What is that?

Reply:
This function no longer exists in imgui I changed it open new Glfw frame and new Imgui frame